### DISCUSSION TITLE
Virtuelle Aktorenkanäle

### DISCUSSION THREAD

**Discussion by chammp** [SukramJ/aiohomematic#430] [General]
Hi Leute, 

ich habe in den letzten Tagen das HmIP-System meinem existierenden HomeAssistant hinzugefügt (RaspberryMatic AddOn und die neue Integration). Ich habe eine Frage zu den virtuellen Aktorenkanälen (vom HmIP-BROLL, um genau zu sein).

Und zwar werden die bei mir aktuell überhaupt nicht in HA exponiert. Ich weiß, dass es da mal eine Diskussion darüber gab, ob/wie die überhaupt als Entitäten erzeugt werden sollen, aber mir ist gerade der Soll-Zustand unklar. Ich würde die jeweiligen Werte der Behanghöhe der virtuellen Kanäle gerne über HA steuern.

Ist weiterhin geplant, diese Entitäten zu erzeugen, oder werden sie eigentlich erzeugt und bei mir ist was kaputt?

LG,
Markus

P.S. Oder bin ich hier falsch und das sollte eigentlich ins Homematic-Forum?

**Comment by SukramJ** [comment]
Das entspricht dem aktuellen Stand, wobei es im Optionsflow keine Optionen mehr gibt:[siehe
](https://github.com/danielperna84/hahomematic/discussions/17#discussioncomment-1692907)

**Reply by chammp** [reply]
OK, das hab ich nur semi-verstanden. Ich weiß, dass es keine Option in der Konfig gibt (aber wohl mal gab), aber _was_ ist denn genau das aktuelle Soll:

- sollten virtuelle Kanäle als versteckte/deaktivierte Entitäten erzeugt werden (-> bei mir ist was kaputt)?
- oder werden virtuelle Kanäle komplett vor HA versteckt und es gibt keinen Weg, das zu ändern (-> dann wäre bei mir nix kaputt und das wäre etwas traurig und würde Dinge eher verkomplizieren)?

**Reply by SukramJ** [reply]
> Bei Cover werden, obwohl vorhanden, keine virtuellen Kanäle angezeigt.

Ist doch eigentlich eindeutig. 
Es werden keine weiteren Entitäten für zusätzliche virtuelle Kanäle erzeugt. Es wird der erste virtuelle Kanal für die Steuerung verwendet (Broll CH4) und die Behanghöhe wird aus dem Sensorkanal (Broll CH3)verwendet.

**Reply by chammp** [reply]
Meine Antwort habe ich geschrieben, bevor dein Link im Posting drin war. Hier ist alles mit Lichtgeschwindigkeit...

Ich denke, die braucht es nicht unbedingt als Entitäten - ich vermute, man kann diese Kanäle dann über den ansprechen...? Das würde meinem Use-Case ja ausreichen (denn am Ende des Tages würde ich lediglich Soll-Werte in die virtuellen Behanghöhen reinschreiben wollen. Auslesen muss ich da nichts und Buttons zur Steuerung sind auch unnötig).

**Reply by SukramJ** [reply]
Dafür  gibt es den generischen Service 
[homematicip_localset_variable_value](https://github.com/danielperna84/custom_homematic#homematicip_localset_variable_value)
